src/Rule.js

```diff
@@ -0,0 +1,77 @@
+class Rule {
+    constructor(){
+        this.findError = [];
+    }
+    isRequired() {
+        this.findError.push(function(value){
+            if(value == undefined)  return {isValid: false, errorMessage: "Ничего не введено"}
```

Comment:
Use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty to check whether property is defined or not